### ISSUE TITLE
[v5] Question: Why do we get missing key messages from stateRecociler

### ISSUE THREAD

**Issue by momosh** [rt2zz/redux-persist#564]
Hey guys, I really like the new v5, awesome job guys.
But I was wondering, why do I get those messages when I code split my reducers? Namely, I inject my reducers on route changes with react-router and I get messages of this sort: 

_redux-persist/stateReconciler: state missing key "repos". state-manager will still store the rehydrated value. If you removed repos from your reducer tree, you should write a migration to remove repos from stored state. If you code-split repos reducer, then this is the expected behavior._

I really don't have a problem with this, since this is only in dev right? But I can't help and wonder if I'm doing something wrong?

**Comment by rt2zz** [comment]
right, that is a valid usage. The goal of this message is to help guide developers who may be unwittingly persisting old data, but at this point I am inclined to remove it, *or* put it behind the `debug` flag. We could add a debug check to the conditional here: https://github.com/rt2zz/redux-persist/blob/master/src/stateReconciler/autoMergeLevel2.js#L19

**Comment by momosh** [comment]
Thanks for a quick answer. Well either is fine, and I really don't mind it, since now I'm sure it won't show in production. And if you want my opinion, I think that debug flag is nice 🙂
